west.yml

```
@@ -112,6 +116,14 @@ manifest:
       repo-path: mbedtls
       revision: mbedtls-2.16.6
       remote: armmbed
+    - name: nanopb
+      path: nanopb
+      revision: nanopb-0.4.2
+      remote: nanopb
+    - name: Alexa-Gadgets-Embedded-Sample-Code
+      path: alexa-embedded
```

based on https://github.com/alexa/Alexa-Gadgets-Embedded-Sample-Code/blob/master/README.md this seems like a better location:
```suggestion
      path: modules/alexa-embedded
```